src/gui/properties/trackersadditiondialog.cpp

```diff
@@ -116,6 +116,8 @@ void TrackersAdditionDialog::torrentListDownloadFinished(const Net::DownloadResu
         BitTorrent::TrackerEntry newTracker {line};
         if (!existingTrackers.contains(newTracker))
         {
+            if(nb != 0)
```

Comment:
Wrong coding style. Should be:
```suggestion
            if (nb != 0)
```

Reply:
Sorry I didn't realise I could commit from here